### PR TITLE
Fixes plus include scripts in ci

### PR DESCRIPTION
**Pull request by kostekIV**
Some adjustments to the scripts to make them work. Also added them to ci that runs on merge to main since they are rather long.

You can check here ci pass: https://github.com/Cardinal-Cryptography/most/actions/runs/9979956139/job/27580250622

### CODE REVIEW

File: .github/workflows/test-transfers.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Runs bridge and perform transfers from and to ethereum.
```
Comment (Marcin-Radecki) [inline .github/workflows/test-transfers.yml:1]: Please add slack notifications as per https://github.com/Cardinal-Cryptography/aleph-node/blob/main/.github/workflows/nightly-fe-e2e-tests.yml#L151-L164 , please use `SLACK_INCOMING_WEBHOOK_MOST_CI_NOTIFICATIONS` secret

**Review by Marcin-Radecki — APPROVED** [review]

File: .github/workflows/test-transfers.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        shell: bash
+        run: |
+          make local-bridgenet
+          sleep 60 # wait 1 minute for everything to be running
+          make deploy-docker
+          DOCKER_RELAYER_COMPILE_CONTRACTS=no-compile make run-relayers
+          
+          make ethereum-azero-transfer
+          sleep 300 # wait 5 minutes for transfer to register on both chains
+          make azero-ethereum-transfer
```
[the remark this answers is not in the document]
Comment (kostekIV) [reply .github/workflows/test-transfers.yml:37]: This test is mostly for checking whether the scripts we use in the repo works as in compiles/runs correctly. THe azero-transfer should fail tho if the eth transfer did not succeed

File: .github/workflows/test-transfers.yml (hunk 1 above)
Comment (kostekIV) [reply .github/workflows/test-transfers.yml:1]: done

**Review by Marcin-Radecki — APPROVED** [review]
Splendid!

**Pull request merged**